lib/struct/logger.ts

```diff
@@ -20,7 +27,14 @@ export class Logger {
       }
 
       await channel.send({
-        content: this.genLogMessage(user, domain, taken, failed),
+        content: await this.genLogMessage(
+          user,
+          domain,
+          taken,
+          channelSentIn.id.toString(),
+          guildId,
+          failed
+        ),
```

Comment:
Please assign the result of `await this.genLogMessage()` to a variable and pass the variable into the `send` function. This is more readable than putting the function call directly in the `send` body.

Reply:
Will do.